js/MVP/containers/DialogmoteContainer.js

```diff
@@ -55,7 +55,9 @@ const DialogmoteContainer = ({ title, breadcrumb = dialogmoteBreadcrumb, display
         {displayTilbakeknapp && <TilbakeknappStyled onClick={browserHistory.goBack} />}
         <BottomInfoStyled>
           <Normaltekst>{texts.bottomText}</Normaltekst>
-          <Lenke>{texts.bottomUrl}</Lenke>
+          <Lenke href="http://www.nav.no/personvern" target="_blank">
```

Comment:
Det er vel litt sånn uu-utfordringer med target blank, men dersom vi ønsker å bruke det så bør man sette på noopener og noreferrer.

ref https://medium.com/@jitbit/target-blank-the-most-underestimated-vulnerability-ever-96e328301f4c

Reply:
Godt poeng, takk for deling!